### ISSUE TITLE
Planning and Reflection

### ISSUE THREAD

**Issue by btomey** [Vidmaster/cybr8420-group4#25]
Overall team planning and individual Contribution

**Comment by btomey** [comment]
We started working on this together to give our group a better sense of how this project needed to be done. After meeting with the professor we found out that we where going into to much depth. What we had done was correct but we where making it to hard on ourselves. As with the last assignment we had to work around our jobs again, this Has been a issue with Bryan as he has had to work late hours multiply time. Bob needed family time over the weekend, he worked hard to get a lot of the claim work done before he left.

**Comment by btomey** [comment]
We started working on this together to give our group a better sense of how this project needed to be done. After meeting with the professor we found out that we were going into to much depth. What we had done was correct but we where making it to hard on our shelves. As with the last assignment we had to work around our jobs again, this has been a issue with Bryan as he has had to work late hours multiply times. Bob needed family time over the weekend, he worked hard to get a lot of the claim work done before he left. Henry also had issue with work this week and is taking care of his injured dog.
